maven-surefire-plugin/src/site/apt/featurematrix.apt.vm

```diff
@@ -27,19 +27,19 @@ Feature Matrix
 
-*---------------------------------------------+-----------+-----------+------------+-----------+----------+
-|| <<Feature>>                                ||<<JUnit3>>||<<JUnit4>>||<<Junit47>>||<<TestNG>>||<<POJO>> |
-*---------------------------------------------+------------+----------+------------+-----------+----------+
-| groups/category support                     |     N      |    N     |      Y     |    Y      |  N       |
-*---------------------------------------------+------------+----------+------------+-----------+----------+
-| security manager support                    |     Y      |    N     |      N     |    N      |  N       |
-*---------------------------------------------+------------+----------+------------+-----------+----------+
-| runOrder support                            |     Y      |    Y     |      Y     |    ?      |  Y       |
-*---------------------------------------------+------------+----------+------------+-----------+----------+
-| run >1 individual test method in a class    |     N      |    Y     |      Y     |    Y      |  N       |
-*---------------------------------------------+------------+----------+------------+-----------+----------+
-| parallel support                            |     N      |    N     |      Y     |    Y      |  N       |
-*---------------------------------------------+------------+----------+------------+-----------+----------+
+*---------------------------------------------+-----------+-----------+------------+-----------+----------+--------------------+
+|| <<Feature>>                                ||<<JUnit3>>||<<JUnit4>>||<<JUnit47>>||<<TestNG>>||<<POJO>> ||<<JUnit Platform>> |
+*---------------------------------------------+------------+----------+------------+-----------+----------+--------------------+
+| groups/category/tags support                |     N      |    N     |      Y     |    Y      |  N       |  Y                 |
+*---------------------------------------------+------------+----------+------------+-----------+----------+--------------------+
+| security manager support                    |     Y      |    N     |      N     |    N      |  N       |  N                 |
+*---------------------------------------------+------------+----------+------------+-----------+----------+--------------------+
+| runOrder support                            |     Y      |    Y     |      Y     |    ?      |  Y       |  N                 |
+*---------------------------------------------+------------+----------+------------+-----------+----------+--------------------+
+| run >1 individual test method in a class    |     N      |    Y     |      Y     |    Y      |  N       |  ?                 |
+*---------------------------------------------+------------+----------+------------+-----------+----------+--------------------+
+| parallel support                            |     N      |    N     |      Y     |    Y      |  N       |  N                 |
```

Comment:
In parallel support we have `N`. Can we utilize ParallelComputer from JUnit4?
The PC covers parallelism. I would like to utilize it because it has a complex possibilities with configuration and I do not want to develop it again.

Reply:
No, `ParallelComputer` cannot be reused. Jupiter currently does not have support for parallel execution, that will be added in https://github.com/junit-team/junit5/issues/60.

Reply:
@marcphilipp 
It's better if we implement PC in surefire. We know what configuration we need. Is there something like `Scheduler` as it was in previous JUnit4? Is there a difference between runner and suite?

Reply:
We should stick with 'N' for now.

Let's see which way is best to implement parallel execution support for the Platform.